xml/ha_maintenance.xml

```diff
@@ -0,0 +1,548 @@
+    To perform maintenance tasks on the cluster nodes, you might need to stop
+    the resources running on that node, to move them, or to shut down or reboot
+    the node.
+<!--Or you may need to stop the cluster software while leaving the services
+   untouched.-->
+    This chapters explains how to manually take down a cluster node without
+    negative side-effects. It also gives an overview of different options the
+    cluster stack provides for executing maintenance tasks.
+   </para>
+  </abstract>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:maintainer/>
+   <dm:status>editing</dm:status>
+   <dm:deadline/>
+   <dm:priority/>
+   <dm:translation>yes</dm:translation>
+   <dm:languages/>
+   <dm:release/>
+   <dm:repository/>
+  </dm:docmanager>
+ </info>
+ <sect1 xml:id="sec.ha.maint.shutdown.node">
+  <title>Implications of Taking Down a Cluster Node</title>
+
+  <para>
+   When you shut down or reboot a cluster node (or stop the &pace; service on a
+   node), the following processes will be triggered:
+  </para>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     The resources that are running on the node will be stopped or moved off
+     the node.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     If stopping the resources should fail or time out, the &stonith; mechanism
+     will fence the node and shut it down.
+    </para>
+   </listitem>
+  </itemizedlist>
+
+  <procedure xml:id="pro.ha.maint.shutdown.node">
+   <title>Manually Rebooting a Cluster Node</title>
+   <para>
+    If your aim is to move the services off the node in an orderly fashion
+    before shutting down or rebooting the node, proceed as follows:
+   </para>
+   <step>
+    <para>
+     On the node you want to reboot or shut down, log in as &rootuser; or
+     equivalent.
+    </para>
+   </step>
+   <step>
+    <para>
+     Put the node into <literal>standby</literal> mode:
+    </para>
+<screen>&prompt.root;crm node standby</screen>
+    <para>
+     That way, services can migrate off the node without being limited by the
+     shutdown timeout of &pace;.
+    </para>
+   </step>
+   <step>
+    <para>
+     Check the cluster status with:
+    </para>
+<screen>&prompt.root;crm status</screen>
+    <para>
+     It shows the respective node in <literal>standby</literal> mode:
+    </para>
+<screen>[...]
+Node <replaceable>&node2;</replaceable>: standby
+[...]</screen>
+   </step>
+   <step>
+    <para>
+     If you have a DLM resource (or other resources depending on DLM), make
+     sure to explicitly stop those resources before stopping the &pace;
+     service:
+    </para>
+<screen>&prompt.crm.res;stop <replaceable>RESOURCE_ID</replaceable></screen>
+    <para>
+     The reason is that stopping &pace; also stops the &corosync; service, on
+     whose membership and messaging services DLM depends. If &corosync; stops,
+     the DLM resource will assume a split-brain scenario and trigger a fencing
+     operation.
+    </para>
```

Comment:
The previous step for  "standby" mode already took care of DLM also:) This step is needed only for the case where the cluster service is going to be shut down while the node/cluster is in maintenance mode.

Reply:
Good, idea - removed this step from the procedure above and added a new procedure at the end of the chapter,  which covers this step and lacks the standby steps.  Fixed with https://github.com/SUSE/doc-sleha/commit/8ce547cc.